hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2406,8 +2432,10 @@ public void setBucketProperty(OmBucketArgs args)
    */
   @Override
   public void deleteBucket(String volume, String bucket) throws IOException {
-    checkAcls(ResourceType.BUCKET, StoreType.OZONE, ACLType.WRITE, volume,
-        bucket, null);
+    if (isAclEnabled) {
```

Comment:
can we move if (isAclEnabled()) inside checkAcls()?

Reply:
This is done to avoid object creation of UGI and InetAddress.
Talked offline, and we agreed on this.